src/IssueViz.UnitTests/Editor/LocationNavigatorTests.cs

```diff
@@ -28,6 +28,7 @@
 using SonarLint.VisualStudio.Integration;
```

Comment:
Please re-review all the tests. The implementation was changed, so I pretty much re-wrote them.